packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartDetails.test.tsx

```diff
@@ -1,3 +1,9 @@
+import { prepareCounterpartOrganization } from '@/components/counterparts/CounterpartDetails/CounterpartForm';
+import {
+  CounterpartContactView,
```

Comment:
Ouch, the side effect of ignoring tests and fixtures from the latest ESLint rules. For now, just remove not used. In next time we need to tune the rules to make them stricter.